config/base.json

```diff
@@ -182,39 +180,6 @@
     "save_checkpoints_steps": 10000,
     "valid_interval": 10000,
     "keep_checkpoint_max": 5,
-    "multi_speaker_training": false, // True: train multi-speaker model; False: training single-speaker model;
```

Comment:
These parameters cannot be deleted because they are being used in TTS and Vocoder. If they are deleted here, but these parameters do not appear in the task-level or model-level config files, bugs may occur.

Reply:
Indeed, the parameters from line 186 to 220 are duplicated.